CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+@Liuhaai#List of owners based on modu
+
+#Default owners
+*		@dustinxie @Liuhaai
+
+consensus/*	@CoderZhi 
+ioctl/* 	@huof6829 
+api/* 		@Liuhaai @millken 
+config/*	@dustinxie @CoderZhi 
+p2p/* 		@Liuhaai @dustinxie
+
+action		@dustinxie
+blockchain	@dustinxie
+blockindex	@dustinxie
+crypto		@dustinxie
+db		@dustinxie
+dispatcher	@dustinxie
+pkg		@dustinxie
+server		@dustinxie
+state		@dustinxie
+actpool		@Liuhaai
```

Comment:
nit question: these does not need /* like above?

Reply:
@dustinxie updated